### PR TITLE
HDDS-2502. Close ScmClient in RatisInsight.

### PR DESCRIPTION
**Pull request by swagle**
## What changes were proposed in this pull request?
Fix sonar issues found in RatisInsight class.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-2502

## How was this patch tested?
Only syntactic changes.

### CODE REVIEW

File: hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/datanode/RatisInsight.java [hunk 1]
```diff
@@ -47,22 +48,24 @@ public RatisInsight(OzoneConfiguration conf) {
...
+          Component dn = new Component(
+              Type.DATANODE, datanode.getUuid().toString(),
+              datanode.getHostName(), 9882);
+          result.add(new LoggerSource(dn, "org.apache.ratis.server.impl",
+              defaultLevel(verbose)));
+        }
       }
     } catch (IOException e) {
-      throw new RuntimeException("Can't enumerate required logs", e);
+      throw new ExceptionInInitializerError(e);
```
Comment (adoroszlai) [inline hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/datanode/RatisInsight.java]: > `ExceptionInInitializerError`: Signals that an unexpected exception has occurred in a static initializer.

Wouldn't `UncheckedIOException` be more appropriate?

Comment (bharatviswa504) [reply hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/datanode/RatisInsight.java]: Ya right, good catch.

Comment (swagle) [reply hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/datanode/RatisInsight.java]: @adoroszlai Agree, much better exception to throw.

**Comment by bharatviswa504** [comment]
Thank You @swagle for the contribution and @adoroszlai for the review.

File: hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/datanode/RatisInsight.java (hunk 1 above)
Comment (bharatviswa504) [reply hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/datanode/RatisInsight.java]: Marking this as resolved, it is updated in the latest commit.

**Pull request merged**